pkg/epinio/models/epinio-resource.js

```diff
@@ -57,4 +59,37 @@ export default class EpinioResource extends Resource {
       throw epinioExceptionToErrorsArray(e);
     }
   }
+
+  async bulkRemove(items, opt = {}) {
+    if ( !opt.url ) {
+      opt.url = (this.links || {})['self'].replace(/\/[^\/]+$/, '?');
```

Comment:
given that this will move up the extend chain out of this location there should be a way to just use `getUrl()`